packages/react-server/core/ReactServerAgent.js

```diff
@@ -10,8 +11,14 @@ function makeRequest (method, url) {
 	return new Request(method, url, API.cache());
 }
 
+const DATA_BUNDLE_CACHE     = {};
```

Comment:
Oh, I see. Can't preload server-side. Still, this variable as a global gives me the heebie-jeebies. Can we add a comment here to that effect as well? I literally almost had a heart attack....